### PR TITLE
Installing Jenkins page redirect

### PR DESCRIPTION
**Pull request by getJv**
issue #2889 - Installing Jenkins page redirect

The wiki url page has double bars (between JENKINS//Installing) so for secure i put two rules targeting the same end point.

### CODE REVIEW

File: dist/profile/templates/confluence/vhost.conf [hunk 1]
```diff
@@ -3003,6 +3003,8 @@ RewriteRule "^/display/JENKINS/CSRF\+Protection$" "https://jenkins.io/doc/book/m
 RewriteRule "^/display/JENKINS/FreeBSD$" "https://jenkins.io/doc/book/installing/#freebsd" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/FreeBSD\+4.9$" "https://jenkins.io/doc/book/installing/#freebsd" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/How\+to\+report\+an\+issue$" "https://www.jenkins.io/participate/report-issue/" [NE,NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS//Installing\+Jenkins$" "https://www.jenkins.io/doc/book/installing/#installing-jenkins" [NE,NC,L,QSA,R=301]
```
Comment (krharsh17) [inline dist/profile/templates/confluence/vhost.conf]:
```suggestion
```
I think as @MarkEWaite mentioned on #1486 you may safely get rid of this rule, as the extra slash ('\\') has been placed mistakenly in the wiki

Comment (getJv) [reply dist/profile/templates/confluence/vhost.conf]: It isn't clear to me yet @krharsh17. Should I keeping use one or two slashes?

Comment (krharsh17) [reply dist/profile/templates/confluence/vhost.conf]: You should use only one slash @getJV

Comment (getJv) [reply dist/profile/templates/confluence/vhost.conf]: Nice! thank you bro. you helped a lot.

**Review by olblak — APPROVED** [review]

**Pull request merged**